pennylane/math/quantum.py

```diff
@@ -247,22 +246,33 @@ def reduce_dm(density_matrix, indices, check_state=False, c_dtype="complex128"):
     return _permute_dense_matrix(density_matrix, sorted(indices), indices, batch_dim)
 
 
-def _batched_partial_trace(density_matrix, indices):
+def partial_trace(matrix, indices, c_dtype="complex128"):
     """Compute the reduced density matrix by tracing out the provided indices.
 
     Args:
-        density_matrix (tensor_like): 3D density matrix tensor. This tensor should be of size
-            ``(batch_dim, 2**N, 2**N)``, for some integer number of wires``N``.
+        matrix (tensor_like): 2D or 3D density matrix tensor. For a 2D tensor, the size is assumed to be
+            ``(batch_dim, 2**n, 2**n)``, for some integer number of wires``n``. In the case of a 3D tensor, the leading dimension is assumed to be a batching dimension
+            ``(batch_dim, 2**n, 2**n)``
+
         indices (list(int)): List of indices to be traced.
 
     Returns:
         tensor_like: (reduced) Density matrix of size ``(batch_dim, 2**len(wires), 2**len(wires))``
 
     **Example**
+    >>> x = np.array([[1, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0]])
+    ...

```

Comment:
why the dots?

Reply:
removed the dots, formatting faux pas on my end